build-tools/packages/build-cli/package.json

```diff
@@ -26,23 +26,21 @@
   ],
   "scripts": {
     "build": "npm run build:genver && concurrently npm:build:compile npm:lint",
-    "build:compile": "npm run tsc",
+    "build:compile": "rimraf dist && npm run tsc",
```

Comment:
Is there a specific reason we have to manually blow away the previous build output?